@udir-design/react/src/tokens/strings.ts

```diff
@@ -0,0 +1,28 @@
+export const labels = {
+  'token-preview': {
+    size: {
+      description:
+        'Verdien til størrelsevariabler settes av `data-size`-attributtet.',
+      'select-label': 'Velg størrelse (data-size)',
+    },
+    color: {
+      description:
+        'Verdien til fargevariabler settes av `data-color`-attributtet.',
+      'select-label': 'Velg farge (data-color)',
+    },
```

Comment:
Markdown-syntaks funker ikkje her, men vi kan gjere sånn? Må rename fila til `strings.tsx`. Merk også forslag til litt meir detaljar om fargevariablane.

```suggestion
    size: {
      description: (
        <>
          Verdien til størrelsevariabler settes av <code>data-size</code>
          -attributtet.
        </>
      ),
      'select-label': 'Velg størrelse (data-size)',
    },
    color: {
      description: (
        <>
          Verdien til disse fargevariablene settes av <code>data-color</code>
          -attributtet. Du kan også bruke fargene direkte, ved å legge til
          fargenavnet etter <code>--ds-color-</code>, f.eks.{' '}
          <code>--ds-color-accent-background-default</code>
        </>
      ),
      'select-label': 'Velg farge (data-color)',
    },
```